src/pytorch_lightning/trainer/trainer.py

```diff
@@ -154,7 +154,7 @@ def __init__(
         fast_dev_run: Union[int, bool] = False,
         accumulate_grad_batches: Optional[Union[int, Dict[int, int]]] = None,
         max_epochs: Optional[int] = None,
-        min_epochs: Optional[int] = None,
+        min_epochs: int = 0,
```

Comment:
I don't know why min_epochs has None as default.
I think it should be zero as default

but I found some test cases that min_epochs is None.
https://github.com/Lightning-AI/lightning/blob/deaadc157b87b653f1c3aa7926a5ff092ab81863/tests/tests_pytorch/trainer/flags/test_min_max_epochs.py#L21
https://github.com/Lightning-AI/lightning/blob/deaadc157b87b653f1c3aa7926a5ff092ab81863/tests/tests_pytorch/callbacks/test_timer.py#L134
Should I put it back?